ft_connectivityanalysis.m

```diff
@@ -221,22 +221,18 @@
 switch cfg.method
   case {'coh' 'csd'}
     if ~isempty(cfg.partchannel)
-      if hasrpt && ~hasjack
+      if hasrpt && ~hasjack && ~isfield(data, 'labelcmb')
         ft_warning('partialisation on single trial observations is not supported, removing trial dimension');
         try
           data = ft_checkdata(data, 'datatype', {'freqmvar' 'freq'}, 'cmbstyle', 'fullfast');
           inparam = 'crsspctrm';
-          hasrpt = 0;
+          hasrpt  = contains(data.dimord, 'rpt');
```

Comment:
can you use `getdimord` instead, or possibly the data dimord that has been determined already earlier in the ft_connectivityanalysis code?